aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -30,7 +31,7 @@ Tensor _add_out(Tensor& out, const Tensor& self, const Tensor& other) {
     cpu_kernel(iter, [&](scalar_t a, scalar_t b) -> scalar_t {
       const auto da = at::dequantize_val(self.q_scale(), self.q_zero_point(), a);
       const auto db = at::dequantize_val(other.q_scale(), other.q_zero_point(), b);
-      float c = da + db;
+      float c = float(da) + db;
```

Comment:
Why is this cast needed? Doesn't dequantize_val already return a float?

 It's probably better to replace `auto` above with the correct types if that assumption is false